Sources/Diagnostics/DiagnosticsEntry.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  DiagnosticsEntry.swift
+//
+//  Created by Cesar de la Vega on 1/4/24.
+
+import Foundation
+
+protocol DiagnosticsEntry: Codable, Equatable {
+
+    var diagnosticType: String { get }
```

Comment:
This can still be removed I think?

Reply:
this is getting removed in the json we send to the backend right?

Reply:
Yup, that's correct, this property won't be read by the backend.